fhir/src/main/java/gov/cms/ab2d/fhir/FhirVersion.java

```diff
@@ -16,25 +16,31 @@
 @Slf4j
 public enum FhirVersion {
     STU3("org.hl7.fhir.dstu3.model", FhirContext.forDstu3(), FhirVersionEnum.DSTU3, "/v1/", "/v1/",
-            org.hl7.fhir.dstu3.model.ResourceType.Patient),
+            org.hl7.fhir.dstu3.model.ResourceType.Patient, false),
```

Comment:
Instead of adding a variable, add a method that for the versions that support default since, handles the behavior.  This is the OO way of doing things by delegating to the instance the instance specific behavior.  For the versions that do not, does the default:

void computeSince(Job job, JobRepo jobRepo);